package/package.json

```diff
@@ -48,6 +48,7 @@
     "@sentry/browser": "^6.1.0",
     "@sentry/react": "^6.2.0",
     "@types/js-cookie": "^3.0.2",
+    "@types/lodash.debounce": "^4.0.7",
```

Comment:
We can remove this and run npm setup again to avoid creating new package lock version